### ISSUE TITLE
Show who read a message

### ISSUE THREAD

**Issue by FantasyCookie17** [Nheko-Reborn/nheko#243]
Currently, unlike Element and Riot, Nheko just shows whether a message was read by *anyone* at all. Having a list of who read it, and least when hovering over the check symbols with the mouse pointer, is what I'd recommend to do. Perhaps also provide a popup menu with a larger list when clicking it.

**Comment by deepbluev7** [comment]
That's planned for when the new event store lands.

**Comment by MasterOfTheTiger** [comment]
I believe nheko supports this by right clicking the check that says if it was read.

**Comment by deepbluev7** [comment]
Yeah, but that checkmark only shows which user read that specific event. If the user read the even after, the read notification may be missing. Which is why we need the new event store, which stores the order of every event :3
